Makefile

```diff
@@ -352,6 +353,10 @@ clean: ## Prune all docker build cache and remove Grapl containers and images
 clean-mount-cache: ## Prune all docker mount cache (used by sccache)
 	docker builder prune --filter type=exec.cachemount
 
+.PHONY: clean-artifacts
+clean-artifacts: ## Remove all dumped artifacts from test runs (see dump_artifacts.py)
+	rm -Rf artifacts_*_[0-9]*
+
```

Comment:
I don't super love that this just keeps adding new folders to <grapl repo root>; I personally would prefer that these be in, like, `/tmp/grapl_artifacts/artifacts_yymmdd_etc`

Reply:
or maybe jsut <grapl_repo_root>/artifacts/yymmdd_etc